app/src/main/java/com/vci/vectorcamapp/core/data/upload/image/ImageUploadWorker.kt

```diff
@@ -558,4 +580,52 @@ class ImageUploadWorker @AssistedInject constructor(
             .build()
         notificationManager.notify(notificationId, notification)
     }
+
+    private suspend fun compressImageFile(uri: Uri, quality: Int) = withContext(Dispatchers.IO) {
+        val resolver = context.contentResolver
+        var bitmap: Bitmap? = null
+        var backupFile: File? = null
+
+        try {
+            backupFile = File.createTempFile("img_backup_", ".tmp", context.cacheDir)
```

Comment:
Question: Will this file naming cause the same issue that we observed with the data upload? Eg. some images not getting compressed